Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -1148,20 +1168,10 @@ final class BuildPlanTests: XCTestCase {
             fileSystem: fileSystem))
 
         let aTarget = try result.target(for: "ATarget").swiftTarget().compileArguments()
-      #if os(macOS)
```

Comment:
These particular ones are testing a higher deployment target for macOS (10.13 vs 10.10) so you'll have to keep the original ifdef in this spot.